code/modules/mob/language.dm

```diff
@@ -393,6 +393,19 @@
 	flags = RESTRICTED | HIVEMIND
 	follow = 1
 
+
+/datum/language/terrorspider
+	name = "TerrorSpider"
+	desc = "Terror spiders have a limited ability to commune over a psychic hivemind, similar to xenomorphs."
+	speech_verb = "hisses"
+	ask_verb = "hisses"
+	exclaim_verb = "hisses"
+	colour = "alien"
+	key = "a"
```

Comment:
What Tigercat said. If another language is to be added, things should generally be unique about it.